website/docs/r/autoscaling_policy.html.markdown

```diff
@@ -39,9 +39,7 @@ resource "aws_autoscaling_group" "bar" {
 }
 ```
 
-## Argument Reference
-
-The following arguments are supported:
+## Arguments Reference
```

Comment:
Should keep the old (singular) header here -- the plurality discrepancy between the two headers is an unfortunate historical one 😅 

```
* website/docs/r/autoscaling_policy.html.markdown: error checking file contents: missing arguments section: ## Argument Reference
```

```suggestion
## Argument Reference
```